transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/util/ImportExportPackageUpdater.java

```diff
@@ -199,7 +269,7 @@ private String addELPackage(String importPkgs) {
 			importPkgs = "";
 		}
 		if (!importPkgs.isEmpty()){
-			importPkgs += ",\n"; //$NON-NLS-1$
+			importPkgs += ",\n "; //$NON-NLS-1$

```

Comment:
yes, package length can be longer (and classname is not part of it)
Manifest files requires to be 72-characters long but they support 512 in fact.
I rely on Eclipse API to write the Manifest file. They are not stripping the content for readability purpose and explains that it should be done during compilation, see https://bugs.eclipse.org/bugs/show_bug.cgi?id=91309
So I would prefer to avoid dealing with this kind of thing in our code.